config/routes.rb

```diff
@@ -9,4 +9,6 @@
   get 'environmental-impact-assessment', to: 'environmental_impact_assessment#index'
   get 'ecosystem-assessment', to: 'ecosystem_assessment#index'
   get 'ecosystem-services', to: 'ecosystem_services#index'
+  get '/download', to: 'metadata#download'
```

Comment:
I would prefer to have `metadata/download` as a path, so it's clear what the download refers to.

Reply:
Thanks I have fixed this too.